.github/workflows/release.yml

```diff
@@ -43,11 +42,10 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: packages
-          path: packages
       - name: Extract ${{ matrix.package }}
         run: |
           mkdir dist
-          mv packages/$(echo ${{ matrix.package }} | sed 's/_/?/')-[0-9]* dist
+          mv */dist/$(echo ${{ matrix.package }} | sed 's/_/?/')-[0-9]* dist
```

Comment:
I think there might be an analogous change missing from publish.rst?

Reply:
After discussion in person, turns out this isn't an issue on publish, because in the publish action, the artefact is coming from the GitHub Release, rather than the CI artefact,